frontend/src/components/JoinGame.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div id="join-game">
+    </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  name: "JoinGame",
+  created: async function() {
+    try {
+      const gameID = this.$route.params.joinID;
+      const response = await axios.post("http://backend:3000/games/" + gameID);
```

Comment:
I assume somehow this calls core.api/add-player
Would it be possible to comment that so it's more understandable for us plebeians?